test/testCLI.ts

```diff
@@ -34,6 +34,18 @@ export function run() {
     )
   })
 
+  test('file in (-i), unreachable definitions flag, pipe out', t => {
+    t.snapshot(
+      execSync('node dist/src/cli.js -i ./test/resources/DefinitionsOnly.json --unreachableDefinitions').toString()
+    )
+  })
+
+  test('file in (-i), style flags, pipe out', t => {
+    t.snapshot(
+      execSync('node dist/src/cli.js -i ./test/resources/DefinitionsOnly.json --style.singleQuote --no-style.semi').toString()
```

Comment:
Please modify `DefinitionsOnly.json` so you're actually testing `style.singleQuote`.